op-interop-mon/flags/flags.go

```diff
@@ -26,13 +26,23 @@ var (
 		EnvVars:  prefixEnvVars("L2_RPCS"),
 		Required: true,
 	}
+
+	// Optional Flags
+	SupervisorEndpointFlag = &cli.StringFlag{
+		Name:     "supervisor-endpoint",
+		Usage:    "The RPC endpoint for the supervisor to call admin_setFailsafeEnabled",
```

Comment:
How about we make this required but use a magic value to disable the failsafe call? As in `SUPERVISOR_ENDPOINT=disabled`.

We could also use "unset" to mean disabled, but I feel like that makes accidental misconfigurations more likely.

Reply:
Another option is to just add a `triggers-failsafe` boolean CLI flag. Pretty clear and explicit and doesn't require any RPC 

Reply:
I think you mean to say it "doesn't require any RPC value changes", because you must *still* restart the service to flip the flag. But it's much cleaner so I'll take it